hw5-10_to_do_list/react_ts/src/components/App/App.tsx

```diff
@@ -0,0 +1,146 @@
+import "./App.css";
+import React, { ChangeEvent, useLayoutEffect, useMemo, useState } from "react";
+import Button from "../Button/Button";
+import AddTaskModalWindow from "../AddTaskModalWindow/AddTaskModalWindow";
+import TaskList from "../TaskList/TaskList";
+import TodayTasksModalWindow from "../TodayTasksModalWindow/TodayTasksModalWindow";
+import TaskRepository from "../../repositories/TaskRepository";
+import Task from "../../types/Task";
+import { datesAreEqual } from "../../utils/Utils";
+import { EntityID } from "../../repositories/Repository";
+import WeatherAPI from "../../api/WeatherAPI";
+
+export default function App() {
+    const [isAddTaskModalWindowShown, setIsAddTaskModalWindowShown] = useState(false);
+    const [initTasks, setInitTasks] = useState<Array<Task>>([]);
+    const [activeTasks, setActiveTasks] = useState(initTasks.filter((t) => !t.isCompleted));
+    const [completedTasks, setCompletedTasks] = useState(initTasks.filter((t) => t.isCompleted));
```

Comment:
you don't need these states. it should be calculatable fields

Reply:
fixed